lms/djangoapps/grades/new/course_grade.py

```diff
@@ -114,7 +121,6 @@ def summary(self):
         grade_summary['totaled_scores'] = self.subsection_grade_totals_by_format
         grade_summary['raw_scores'] = list(self.locations_to_weighted_scores.itervalues())
 
-        self._log_event(log.warning, u"grade_summary, percent: {0}, grade: {1}".format(self.percent, self.letter_grade))
```

Comment:
The percent and letter_grade data is now logged up in `grade_value`, which only fires on updates instead of on access.